src/osd/ReplicatedBackend.cc

```diff
@@ -715,13 +715,13 @@ void ReplicatedBackend::sub_op_modify_reply(OpRequestRef op)
       }
     } else {
       assert(ip_op.waiting_for_applied.count(from));
+      ip_op.waiting_for_applied.erase(from);
```

Comment:
@tchaikov sub_op_modify_commit, sub_op_modify_applied send reply separately,so this should not happen.